google-cloud-spanner/lib/google/cloud/spanner/client.rb

```diff
@@ -2159,7 +2198,9 @@ def batch_create_sessions session_count
             session_count,
             labels: @session_labels,
             database_role: @database_role
-          resp.session.map { |grpc| Session.from_grpc grpc, @project.service, query_options: @query_options }
+          resp.session.map do |grpc|
```

Comment:
I guess this is not a change related to directed reads right?

Reply:
ya just a styling change, does not affect the behaviour